### PR TITLE
Render contact custom properties on home page

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/165035/hubspot-contact-field-mapping

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -53,7 +55,7 @@ function App() {
         },
     }, [client, unlinkContactFn]);
 
-    if (!client || isLoading) {
+    if (!client || isLoading && !context) {
```
Comment (HappyPaul55) [inline src/App.tsx]: Do we need the context check? If `isLoading`, I'd expect a spinner?

File: src/components/common/Builder/BlocksBuilder/utils/validateConfig.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { isEmpty } from "lodash-es";
```
Comment (HappyPaul55) [inline src/components/common/Builder/BlocksBuilder/utils/validateConfig.ts]: No need for isEmpty, `arr.length === 0` is good enough in this context/file.

File: src/components/common/Tag/Tag.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      backgroundColor: `${lightTheme.colors.grey80}33`,
+      textColor: lightTheme.colors.grey80,
+    }}
+    label={tag}
+    withClose={false}
+  />
+);
+
+const Tags: FC<{ tags: string[] }> = ({ tags }) => {
+  if (!Array.isArray(tags) || !size(tags)) {
```
Comment (HappyPaul55) [inline src/components/common/Tag/Tag.tsx]: No need for `lodash` here, `tags.length === 0` will work here.

**Review by HappyPaul55 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (zpawn) [reply src/App.tsx]: Removed _(sorry, I removed but forgot to comment)_

File: src/components/common/Builder/BlocksBuilder/utils/validateConfig.ts (hunk 2 above)
Comment (zpawn) [reply src/components/common/Builder/BlocksBuilder/utils/validateConfig.ts]: removed

File: src/components/common/Tag/Tag.tsx (hunk 3 above)
Comment (zpawn) [reply src/components/common/Tag/Tag.tsx]: removed

File: src/components/common/Builder/BlocksBuilder/GenerateBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import type { ComponentType } from "react";
+
+type Props<Meta> = {
+  meta: Meta;
+  Component: ComponentType<{ value: unknown } & Record<string, unknown>>;
+  value: unknown;
+};
+
+const GenerateBlock = <Meta,>({ meta, Component, value }: Props<Meta>) => {
+  const blockType = get(meta, ["type"]);
```
Comment (HappyPaul55) [inline src/components/common/Builder/BlocksBuilder/GenerateBlock.tsx]: No need for `lodash` here, `meta?.type` is fine.

**Review by HappyPaul55 — APPROVED** [review]
Ideally we should be using alias to avoid many `../` but for now approved.

**Pull request merged**